resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -367,6 +368,7 @@ const jobToValues = (job: Job | null, locale: string): JobFormValues =>
         securityLevel: job.security_clearance_id || "",
         language: job.language_requirement_id || "",
         city: job[locale].city || "",
+        education: job[locale].education || "",
```

Comment:
Education isn't actually part of the form, so it doesn't really need to be set in JobToValues or UpdateJobWithValues

Reply:
What if the form is being used to update an existing job poster? I thought in that case the job information should already be there and passed through.

Reply:
Yeah, now that I take a step back and think about it. It doesn't make sense. 👍